avaframe/com4FlowPy/flowCore.py

```diff
@@ -479,9 +504,13 @@ def calculation(args):
         startcell_idx += 1
     # end = datetime.now().replace(microsecond=0)
     gc.collect()
-    return z_delta_array, flux_array, count_array, z_delta_sum, backcalc, fp_travelangle_array, \
+    if forestInteraction:
```

Comment:
names of all return variables must be updated to match the new variable names (```zDeltaArray, travelLengthArray, ...```)